api/Sources/Api/PairQL/Dashboard/Pairs/RequestPublicKeychain.swift

```diff
@@ -0,0 +1,51 @@
+import Dependencies
+import PairQL
+
+struct RequestPublicKeychain: Pair {
+  static let auth: ClientAuth = .admin
+
+  struct Input: PairInput {
+    var searchQuery: String
+    var description: String
+  }
+}
+
+extension RequestPublicKeychain: Resolver {
+  static func resolve(with input: Input, in context: AdminContext) async throws -> PairQL
+    .SuccessOutput {
```

Comment:
rather than use `PairQL.SuccessOutput` (probably the lsp filled it in) i prefer to just use the associated type `Output` in these type signatures